fe/fe-core/src/main/java/com/starrocks/catalog/Catalog.java

```diff
@@ -3045,13 +3046,18 @@ public void createTableLike(CreateTableLikeStmt stmt) throws DdlException {
             } finally {
                 db.readUnlock();
             }
-            CreateTableStmt parsedCreateTableStmt =
-                    (CreateTableStmt) SqlParserUtils.parseAndAnalyzeStmt(createTableStmt.get(0), ConnectContext.get());
-            parsedCreateTableStmt.setTableName(stmt.getTableName());
-            if (stmt.isSetIfNotExists()) {
-                parsedCreateTableStmt.setIfNotExists();
+            StatementBase statementBase = SqlParserUtils.parseAndAnalyzeStmt(createTableStmt.get(0), ConnectContext.get());
+            if (statementBase instanceof CreateTableStmt) {
+                CreateTableStmt parsedCreateTableStmt =
+                        (CreateTableStmt) SqlParserUtils.parseAndAnalyzeStmt(createTableStmt.get(0), ConnectContext.get());
+                parsedCreateTableStmt.setTableName(stmt.getTableName());
+                if (stmt.isSetIfNotExists()) {
```

Comment:
how about create view like table?

Reply:
create view like table is currently not supported.